tetris.sh

```diff
@@ -17,13 +17,60 @@
 #
 # Author: Kirill Timofeev <kt97679@gmail.com>
 #
+# Localized Support: Rojen Zaman <rojen@riseup.net> | lang/README.md
+#
 # This program is free software. It comes without any warranty, to the extent
 # permitted by applicable law. You can redistribute it and/or modify it under
 # the terms of the Do What The Fuck You Want To Public License, Version 2, as
 # published by Sam Hocevar. See http://www.wtfpl.net/ for more details.
 
 set -u # non initialized variable is an error
 
+# show help page when given -h argument
+function help_page() {
+cat <<EOF
```

Comment:
Let's think how we can preserve indentation in the function

Reply:
so, you say such as:

```bash
usage() {

cat <<EOF
usage: ./`basename $0` -l <language>
supported languages:
english (it is defualt, option not required)
$(ls -I README.md -1t lang/ | cut -d'.' -f1)    
EOF
exit 0;

}
```

Reply:
sorry, I was not clear, I was thinking about something like:
```
usage() {
    printf "%s\n" \
        "usage: ./$(basename $0) -l <language>" \
        "supported languages:" \
        "english (it is defualt, option not required)" \
        "$(ls -I README.md -1t lang/ | cut -d'.' -f1)"
    exit 0;
}
```

Reply:
uh okay. i get it

I thought you wanted columns and I was struggled in that, luckily not :)